### PR TITLE
redid ram object

### PR DESCRIPTION
**Pull request by Graur**
#684

### CODE REVIEW

**Comment by Graur** [comment]
@yegor256 please, have a look

**Comment by yegor256** [comment]
@Graur why can't we make this `ram-slice` object a kid of `ram`?

**Comment by Graur** [comment]
@yegor256 Sure! I did it, thanks! Please, have a look

**Review by yegor256 — CHANGES_REQUESTED** [review]
@Graur see above

File: eo-runtime/src/main/eo/org/eolang/ram.eo [hunk 1]
```diff
@@ -20,17 +20,26 @@
...
-  # Write bytes B at position P.
-  # Returns itself.
-  [p b] > write /true
+  # Write bytes from position.
+  [position data] > write /true
+
+  # Slice fixed size of bytes from position.
+  [position size] > slice /ram-slice
+
+  [rho position size] > ram-slice
```
[the remark this answers is not in the document]
Comment (Graur) [reply eo-runtime/src/main/eo/org/eolang/ram.eo:41]: @yegor256 I have got a problem with getting `ram-slice` parent object. The chain of parents `ram -> slice -> ram-slice -> @` doesn't work as I expected

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**